erpnext_documentation/www/docs/user/manual/en/education/student-leave-application.md

```diff
@@ -1,23 +1,42 @@
 <!-- add-breadcrumbs -->
-# Student Leave Application 
+# Student Leave Application
 
-**Leave Application will have a record of the leaves applied by the students.**
+**Student Leave Application is a formal document to keep a track of leaves for a student.**
 
-To access Leave Application, go to:
+To access Student Leave Application list, go to:
 
 > Home > Education > Attendance > Student Leave Application
 
-To create a Student Leave application record, enter the **Student** and the date for which the leave is applied and save.
+## 1 How to create a Student Leave Application
 
-![Student Leave Application](/docs/assets/img/education/education-student-leave-application.gif)
+1. Go to the Student Leave Application list, and click on New.
+2. Select the Student.
+3. Set the 'From Date' and 'To Date' fields for specifying the period.
+4. Marking Attendance for the Leave Application is governed by the 'Attendance Based On' field. In ERPNext, Student Attendance can be marked in two ways:
 
-In case the student is not attending the institute to participate or represent institute in any event, he/she can be marked as present from the Leave Application itself. 
+    * **Course Schedule**: If attendance is taken for every lecture (in colleges/universities), then the leave application can be created for that particular course schedule slot.
 
-Once a Leave Application is recorded for a student it will not be recorded in the absent student report as he has applied for a leave. 
+    * **Student Group**: If attendance is taken for the entire day then student group (class/division) is used to mark attendance so leave is for the entire day
```

Comment:
```suggestion
    * **Student Group**: If attendance is taken for the entire day then student group (class/division) is used to mark attendance so that leave is calculated for the entire day
```